src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1191,7 +1193,12 @@ int xcldev::device::runTestCase(const std::string& py,
         auto device = pcidev::get_dev(m_idx);
         std::string bdf = boost::str(boost::format("%04x:%02x:%02x.%01x") % device->domain % device->bus % device->dev % device->func);
 
-        cmd = xrtTestCasePath + " " + xclbinPath + " -d " + bdf;
+        // Other new flow test cases are using OpenCL APIs, which can find device by BDF.
+        // If XRT and XCL APIs support open device by BDF, we can unify below two cmds.
+        if (py.find("xrt_iops_test") || py.find("xcl_iops_test"))
+            cmd = xrtTestCasePath + " " + xclbinPath + " -d " + std::to_string(m_idx) + " " + args;
```

Comment:
we have to support opening device by bdf, m_idx may point to a different device in iops test process.

Reply:
Fixed.